sjb/actions/pull_request_sync.py

```diff
@@ -77,7 +77,7 @@ class PullRequestSyncAction(Action):
 
     def __init__(self, repository):
         self.repository = repository
-        self.parent_sync = SyncAction(repository)
+        self.parent_sync = SyncAction(repository, repository)
```

Comment:
Can we use `dependency_repository=repository` to explain what this is here please